Source/Core/VideoCommon/VideoConfig.cpp

```diff
@@ -93,9 +93,20 @@ void VideoConfig::Load(const std::string& ini_file)
 	enhancements->Get("PostProcessingShader", &sPostProcessingShader, "");
 	enhancements->Get("StereoMode", &iStereoMode, 0);
 	enhancements->Get("StereoDepth", &iStereoDepth, 20);
-	enhancements->Get("StereoConvergence", &iStereoConvergence, 20);
+	enhancements->Get("StereoConvergencePercentage", &iStereoConvergencePercentage, 100);
 	enhancements->Get("StereoSwapEyes", &bStereoSwapEyes, false);
 
+	// Some users still have absolute convergence values in their config,
+	// convert those to percentage values.
+	int old_convergence;
+	enhancements->Get("StereoConvergence", &old_convergence, -1);
+	if (old_convergence != -1)
+	{
+		iStereoConvergencePercentage = (old_convergence * 100) / 20;
+		enhancements->Set("StereoConvergencePercentage", iStereoConvergencePercentage);
+		enhancements->Delete("StereoConvergence");
```

Comment:
Oops, forgot about that, like I said this is probably not the right place for it. Do we have any place where we can convert old settings?